tests/test_formatters.py

```
@@ -206,524 +206,526 @@
-        # Errors
-        self.assertRaises(ValueError, format_day_sec, 0, 0., order='YMD')
-        self.assertRaises(ValueError, format_day_sec, 0, 0., kind='X')
-        self.assertRaises(ValueError, format_day_sec, 0, 0.,
-                                                      buffer=np.empty((3,), dtype='int'))
-        self.assertRaises(ValueError, format_day_sec, [[0,0],[0,0]], 0.,
-                                                      buffer=np.empty((3,3), dtype='U40'))
+        # Check a case with the exact right buffer size so no "extra"
+        result = format_day_sec(0, 43201, buffer=None)
+        assert result == '2000-01-01T12:00:01'
+
+        for l in (len(result), len(result)+1):
```

Ruff (0.15.6)

[error] Ambiguous variable name: `l`

(E741)